rpcs3/Emu/RSX/GSRender.h

```diff
@@ -18,6 +18,8 @@ enum wm_event
 
 class GSRender : public rsx::thread
 {
+	bool m_showed_window = false;
```

Comment:
m_window_visible

Reply:
This variable is supposed to check if the window was shown programmatically once.
so calling it "visible" is false.
Remember, the user can hide it themselves